archivesunleashed/auk
# Adds Notebook Learning Guide, Resolves #311

**GitHub issue(s)**:

#311 

# What does this Pull Request do?

This adds a new learning guide: "Getting Started with the Archives Unleashed Cloud Jupyter Notebooks," written by Sarah McTavish.

In addition, this PR:
* Adds a link to the learning guide from the derivatives landing page;
* Adds links to it to complement the blog post written by Ryan Deschamps; and
* Adds a test for it.

# How should this be tested?

TravisCI should turn green.

*More importantly,* others should give this a read and make sure it properly addresses how to install the Notebooks. I'll take the lead on updating the text and can chat with Sarah as needed.

# Interested parties

@ruebot @SamFritz
